src/OrchardCore/OrchardCore/Modules/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -272,6 +273,71 @@ private static void AddAntiForgery(OrchardCoreBuilder builder)
             });
         }
 
+        /// <summary>
+        /// Adds backwards compatibility to the handling of SameSite cookies.
+        /// </summary>
+        public static void AddSameSiteCookieBackwardsCompatibility(OrchardCoreBuilder builder)
+        {
+            builder.ConfigureServices(services =>
+                {
+                    services.Configure<CookiePolicyOptions>(options =>
+                    {
+                        options.MinimumSameSitePolicy = AspNetCore.Http.SameSiteMode.Unspecified;
+                        options.OnAppendCookie = cookieContext => CheckSameSiteBackwardsCompatiblity(cookieContext.Context, cookieContext.CookieOptions);
+                        options.OnDeleteCookie = cookieContext => CheckSameSiteBackwardsCompatiblity(cookieContext.Context, cookieContext.CookieOptions);
+                    });
+                })
+                .Configure(app =>
+                {
+                    app.UseCookiePolicy();
+                });
+        }
+
+        private static void CheckSameSiteBackwardsCompatiblity(HttpContext httpContext, CookieOptions options)
+        {
+            var userAgent = httpContext.Request.Headers["User-Agent"].ToString();
+
+            if (options.SameSite == AspNetCore.Http.SameSiteMode.None)
```

Comment:
You need moar `using` loooooove in your life!

Reply:
Will add some love.

Reply:
No can't do, it conflicts with another namespace. So you are stuck with it.